### ISSUE TITLE
Issue with  Where-Object $Matches return for Mount-Path

### ISSUE THREAD

**Issue by MatthewFallon** [tillig/ps-bash-completions#11]
Creating an issue for discussion to go with my pull request. Tried installing this module today and found it not working on my new machine, namely the root mount path it was receiving `C:` rather than the expected `/` or `/mnt/`. 

I diagnosed it as due to `$Matches` only being resolved to its first match in its statement in https://github.com/tillig/ps-bash-completions/blob/8143310c7f51f58223491a411873f34f96cbf047/PSBashCompletions/Get-MountPath.ps1#L14 rather than the second match as seems to be intended. Evidence of issue:

```ps1
PS C:\Users\mfgin> Register-BashArgumentCompleter "kubectl" C:\completions\kubectl_completions.sh -Verbose
VERBOSE: Found bash in path.
VERBOSE: bash = C:\Windows\system32\bash.exe
VERBOSE: Starting command completion registration for kubectl
VERBOSE: Completion bridge = C:c/Users/mfgin/Documents/PowerShell/Modules/PSBashCompletions/1.2.4/bash_completion_bridge.sh
VERBOSE: kubectl is not an alias.
VERBOSE: Bash completions for kubectl = C:c/completions/kubectl_completions.sh
VERBOSE: Completion command = &"C:\Windows\system32\bash.exe" "C:c/Users/mfgin/Documents/PowerShell/Modules/PSBashCompletions/1.2.4/bash_completion_bridge.sh" "C:c/completions/kubectl_completions.sh" "<url-encoded-command-line>"
```

versions of Powershell seemingly affected:
```ps1
PS C:\Users\mfgin\Development\git> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      7.4.1
```
```ps1
PS C:\Users\mfgin> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      5.1.22621.2506
```

Adding a pull request of the updated fix in a moment. Just want an issue to reference if discussion is needed 😄

**Comment by tillig** [comment]
Can you provide the output of 'bash -c mount' ? If you look through the issues list you'll see that trying to parse the mount output has been one of the larger pain points here.

**Comment by tillig** [comment]
Actually, it might be good if you add your mount output to your PR as a test so we make sure it fixes the issue.

**Comment by tillig** [comment]
Merged and released - v1.2.5. Thanks!
